config/repo-values.yaml

```diff
@@ -227,7 +227,7 @@ sync:
     - name: jenkins
       url: https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/chart
     - name: talend
-      url: https://talend.github.io/helm-charts-public
+      url: https://talend.github.io/helm-charts-public/stable
```

Comment:
At the root of the repository there is a `repos.yaml` file that needs to be updated as well. Can you change it there, too.

Reply:
Hi. As you can see (https://github.com/helm/hub/blob/master/repos.yaml#L618) there is no need for an update in `repos.yaml`: already the proper value. The whole point of this PR is to fix the previous PR we did, forgotting to provide the new repo URL value in `repo-values.yaml`.
Thanks.

Reply:
@mattfarina yes, that was a mistake of mine, I forgot to update this file, so this PR is just a fix of it. Sorry about that.

Reply:
I see. oops. Thanks for fixing this.